src/swagger.yml

```diff
@@ -2,29 +2,46 @@ swagger: '2.0'
 info:
   description: RESTful API for the Denner Portal
   version: 1.0.0
-  title: Denner Portal 2.0
+  # Caution: For development imports in API Gateway use your personal name acronym as version, i.e. "denner-api-iw"
+  title: "denner-api-cd"
   contact:
     name: DETAIL NET
     email: support+denner-api@detailnet.ch
 host: denner-api.detailnet.ch
 basePath: /v1
+x-detail-stage-variables:
+  prod:
+   - lambda_env: "prod"
+   - lambda_alias: "live"
+   - url_advertising: "denner-adv-new.frb.io/api"
+   - url_assets: "denner-ats-new.frb.io/api"
+   - url_articles: "denner-art-new.frb.io/api"
+   - url_stores: "denner-sts-new.frb.io/api"
+  test:
+   - lambda_env: "test"
+   - lambda_alias: "live"
+   - url_advertising: "denner-advertising-test.detailnet.ch/api"
+   - url_assets: "denner-assets-test.detailnet.ch/api"
+   - url_articles: "denner-articles-test.detailnet.ch/api"
+   - url_stores: "denner-stores-test.detailnet.ch/api"
 schemes:
   - https
-consumes:
-  - application/json
-produces:
-  - application/json
 securityDefinitions:
-  appId:
-    type: apiKey
-    description: Application identifier
-    name: App-ID
-    in: header
   appKey:
     type: apiKey
     description: Application key
     name: App-Key
     in: header
+    x-amazon-apigateway-authtype: "custom"
+    x-amazon-apigateway-authorizer:
+      authorizerUri: "arn:aws:apigateway:eu-west-1:lambda:path/2015-03-31/functions/arn:aws:lambda:eu-west-1:802555642192:function:denner-lambda-tools_authorizeApiRequest:live/invocations"
+      authorizerCredentials: "arn:aws:iam::802555642192:role/aws-apigateway-role.denner-api"
+      authorizerResultTtlInSeconds: 3600 #todo Check this, in the other authorizers this is set to 300
+      type: "token"
+x-amazon-apigateway-request-validators:
```

Comment:
@ivan-wolf This is something new for me, it has to do with 

![image](https://user-images.githubusercontent.com/7221909/47513136-e76d1f00-d87d-11e8-9bf8-9bca1376ea41.png)

That I've introduced in my  test API (see image after)